src/migrations/20230919103955-create-feature-environments-metrics-table.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+        CREATE TABLE IF NOT EXISTS feature_environments_metrics(
+           environment  varchar(100) not null
+               references environments(name)
+                   on delete cascade,
+           feature_name varchar(255) not null
+               references features(name)
+                   on delete cascade,
+           last_seen_at timestamp with time zone,
+           primary key (environment, feature_name)
+        );
+
+        INSERT INTO feature_environments_metrics (environment, feature_name, last_seen_at)
+        SELECT
+            feature_environments.environment,
+            feature_environments.feature_name,
+            feature_environments.last_seen_at
+        FROM feature_environments
+        ON CONFLICT (environment, feature_name)
+            DO NOTHING;
+
+        DROP VIEW features_view;
+
+        CREATE VIEW features_view AS
+        SELECT features.name                     as name,
+               features.description              as description,
+               features.type                     as type,
+               features.project                  as project,
+               features.stale                    as stale,
+               features.impression_data          as impression_data,
+               features.created_at               as created_at,
+               features.archived_at              as archived_at,
+               features.last_seen_at             as last_seen_at,
+               feature_environments_metrics.last_seen_at as env_last_seen_at,
+               feature_environments.enabled      as enabled,
+               feature_environments.environment  as environment,
+               feature_environments.variants     as variants,
+               environments.name                 as environment_name,
+               environments.type                 as environment_type,
+               environments.sort_order           as environment_sort_order,
+               feature_strategies.id             as strategy_id,
+               feature_strategies.strategy_name  as strategy_name,
+               feature_strategies.parameters     as parameters,
+               feature_strategies.constraints    as constraints,
+               feature_strategies.sort_order     as sort_order,
+               fss.segment_id                    as segments,
+               feature_strategies.title          as strategy_title,
+               feature_strategies.disabled       as strategy_disabled,
+               feature_strategies.variants       as strategy_variants
+        FROM features
```

Comment:
What is all this needed for?

Reply:
drops and recreates the featureView (I found out the hard way that this is much easier than modifying_ - changing where the env_last_seen is being read.

Reply:
``` feature_environments_metrics.last_seen_at as env_last_seen_at,```